sdk/servicebus/azure-servicebus/swagger/servicebus-swagger.json

```diff
@@ -327,15 +371,15 @@
         "DefaultMessageTimeToLive": {
           "description": "ISO 8601 default message timespan to live value. This is the duration after which the message expires, starting from when the message is sent to Service Bus. This is the default value used when TimeToLive is not set on a message itself.",
           "type": "string",
-          "format": "duration",
+          "format": "iso8601",
           "xml": {
             "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
           }
         },
         "DuplicateDetectionHistoryTimeWindow": {
           "description": "ISO 8601 timeSpan structure that defines the duration of the duplicate detection history. The default value is 10 minutes.",
           "type": "string",
-          "format": "duration",
+          "format": "iso8601",
```

Comment:
What data type does Java generate for "iso8601"? Python generates it as a str. 
If it's "duration", Python generates it as a datetime.timedelta, which is the right data type for a time period in Python.

Reply:
If you look at the spec, format is an open value. It can offer a serialization hint. This value isn't a date-time.
https://swagger.io/docs/specification/data-models/data-types/

> However, format is an open value, so you can use any formats, even not those defined by the OpenAPI Specification, such as:

I can remove it for now, since AutoRest doesn't do anything with it.

Reply:
yes format is open value. I guess it's useful to Autorest. The Python code generated uses type datetime.timedelta if format is "duration". So could you put "duration" here? 
What data type does the generated Java code use?

Reply:
I can generate timedelta now.